src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -75,7 +72,17 @@
     /**
      * Digest function that is used to compute the kubernetes label "jenkins/label-digest"
      */
-    public static final HashFunction LABEL_DIGEST_FUNCTION = Hashing.sha1();
+    protected static final MessageDigest LABEL_DIGEST_FUNCTION;
+    static {
+        try {
+            LABEL_DIGEST_FUNCTION = MessageDigest.getInstance("SHA-256");
+        } catch (NoSuchAlgorithmException e) {
+            // will never happen, SHA-256 support required on every Java implementation
+            e.printStackTrace();
+            // throw runtime exception to allow variable to be set as final
+            throw new RuntimeException(e);
```

Comment:
Why not `throw new AssertionError(e);` like the previous impl?

Reply:
fixed in ff1d11f